client/me/help/controller.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
 		analytics.pageView.record( basePath, 'Help' );
 
 		renderWithReduxStore(
-			React.createElement( Help ),
+			<Help isCoursesEnabled={ config.isEnabled( 'help/courses' ) }/>,
```

Comment:
Should we go ahead and abstract this out into a function or something, so that it's easy later to plug in more rules (like one to display it only for eligible users)?